### PR TITLE
adding gitops-days-2022-recap blog

### PR DESCRIPTION
**Pull request by staceypotter**
Adding GitOps Days 2022 recap blog. :)

### CODE REVIEW

File: content/blog/gitops-days-2022/index.mdx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title: GitOpsCon North America 2021
```
Comment (scottrigby) [inline content/blog/gitops-days-2022/index.mdx:2]: Fix

```suggestion
title: GitOps Days 2022
```

**Review by scottrigby — CHANGES_REQUESTED** [review]
title typo, otherwise lgtm!

**Comment by scottrigby** [comment]
@staceypotter Also, re the event on https://opengitops.dev/events, should the recordings link go to https://www.gitopsdays.com/ or a youtube playlist at this point?

**Comment by staceypotter** [comment]
@scottrigby yes the events page should link to the playlist now. Also, I'm not sure why my DCO signoff isn't working now, investigating. Thanks for the review and typo catch!

**Review by scottrigby — APPROVED** [review]
Looks good 👍

**Comment by scottrigby** [comment]
Ps playlist link can be updated in separate PR 👌

**Pull request merged**